CONTRIBUTING.md

```diff
@@ -210,10 +210,10 @@ accordingly.
 
 ```bash
 # Clone into the repo.
-git clone git@github.com:google/openhtf.git
+git clone https://github.com/google/openhtf.git
```

Comment:
Curious what the motivation is for changing the recommended way from ssh to https?

Reply:
I get an error when using ssh:
Permission denied (publickey).
fatal: Could not read from remote repository.

Which is probably fixable.  

The github page specifies  "Clone with HTTPS" as the default option which works, so I figured this would be the method to get someone up and running quicker.